shared/Rakefile

```diff
@@ -65,17 +65,31 @@ end
 
 require "rake/testtask"
 desc "Run functional tests for all custom protos"
-Rake::TestTask.new do |t|
-  t.libs << "test"
-  t.test_files = FileList["test/**/*_test.rb"]
-  t.warning = false
+task :test do
+  Rake::Task["test:showcase"].invoke
 end
 namespace :test do
   desc "Run functional tests for showcase protos"
   Rake::TestTask.new :showcase do |t|
-    t.libs << "test"
+    require "tmpdir"
+
+    client_lib = Dir.mktmpdir
```

Comment:
I think creating the temp directory, running the `protoc` command, and cleaning up the temp directory could all be done in the test helper. The test helper already starts and stops the echo server. I don't think this _needs_ to be performed in the rake task.

Reply:
Yeah, that's true. I'll move it.